packages/sources/crypto-api-v2/src/endpoint/price.ts

```diff
@@ -0,0 +1,42 @@
+import { Requester, Validator } from '@chainlink/ea-bootstrap'
+import { ExecuteWithConfig, Config } from '@chainlink/types'
+
+export const Name = 'price'
+
+interface ResponseSchema {
```

Comment:
We should probably export these as well. Just a good habit so we could import these types in other composite adapters in the future.